modules/custom-status/meta/required-fields-cron-cleaner.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * class ReqMetadataFieldsCronCleaner
+ * Cleans up fields that are required on a custom status, and should be cleaned up.
+ */
+
+namespace VIPWorkflow\Modules\CustomStatus\Meta;
+
+use VIPWorkflow\Modules\Custom_Status;
+use VIPWorkflow\VIP_Workflow;
+
+class RequiredFieldsCronCleaner {
```

Comment:
Common utility class that handles the cleanup of the required fields without needing to duplicate the code everytime. This would be an ideal candidate for a cron cleaner as well imo.

Reply:
I see the use of `Cron` in the name here, which I would take to mean there's scheduled cron calls. I'm not seeing anything cron-related though, am I missing something?

Reply:
I added it on purpose because we can add cron to this down the line, to handle data cleanup. I meant for this to be a central cleaner of data I suppose.